api/admin/controller.py

```diff
@@ -1636,6 +1636,7 @@ def metadata_services(self, do_get=HTTP.get_with_timeout,
                 service, do_get=do_get, do_post=do_post, key=key
             )
             if problem_detail:
+                self._db.rollback()
```

Comment:
I noticed this while testing - if you're setting up a Metadata Wrangler integration and registering fails, we should rollback the creation of that integration so you can try the registration again.

Reply:
Not for this branch, but it would also be useful to see the problem detail document the metadata wrangler sent.